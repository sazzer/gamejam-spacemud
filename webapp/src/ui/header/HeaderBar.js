import React from 'react';
import { Menu, Item } from 'semantic-ui-react';
import { Interpolate } from 'react-i18next';

/**
 * The Header Bar for the entire application
 * @constructor
 */
export function HeaderBar() {
    return (
        <Menu attached borderless className="spacemud_header">
            <Item className="header">
                <Interpolate i18nKey="page.header" />
            </Item>

            <Menu inverted compact className="right">
            </Menu>
        </Menu>
    );
}
