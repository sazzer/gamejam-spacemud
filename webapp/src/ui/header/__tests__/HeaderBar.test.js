import React from 'react';
import { shallow } from 'enzyme';
import { HeaderBar } from '../HeaderBar';

/**
 * Set up the component being tested
 */
function setup({} = {}) {
    const wrapper = shallow(<HeaderBar />);

    return {
        wrapper
    };
}

describe('Rendering', () => {
    it('renders the expected HTML', () => {
        const { wrapper } = setup({});

        expect(wrapper).toMatchSnapshot()
    });
});
