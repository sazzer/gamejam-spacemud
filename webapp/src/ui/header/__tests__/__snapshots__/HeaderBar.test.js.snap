// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering renders the expected HTML 1`] = `
<Menu
  attached={true}
  borderless={true}
  className="spacemud_header"
>
  <Item
    className="header"
  >
    <Interpolate
      className=""
      i18nKey="page.header"
    />
  </Item>
  <Menu
    className="right"
    compact={true}
    inverted={true}
  />
</Menu>
`;
